packages/workspace/src/parser/internal/native/parse.ts

```
@@ -102,6 +103,15 @@ export async function parseBuffer(
           TOKEN_TYPES.MERGE_CONFLICT,
         ))
       }
+    } else if (e instanceof UnknownCharacter) {
```

do we need another "general" catch here so we don't completely ignore errors like we did with this case before this fix?
seems like the code already did throw an error from `consumeValue` and this code would just silently suppress it.
maybe we should at least move the "unknown error" log to here, just in case the next time won't come from `consumeValue` but from somewhere else?